frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineRunItem.tsx

```diff
@@ -30,7 +30,7 @@ const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
           {lastUpdated && <span className="text-muted">&nbsp;({fromNow(lastUpdated)})</span>}
         </GridItem>
         <GridItem span={3}>
-          <Status status={pipelineRunFilterReducer(pipelineRun)} />
+          <Status status={pipelineRunStatus(pipelineRun) || 'Pending'} />
```

Comment:
Rare instances it wouldn't start immediately and showed `-` which I felt could be better shown as `Pending`.